client/src/app/+videos/+video-edit/video-add-components/video-upload.component.ts

```diff
@@ -67,7 +67,7 @@ export class VideoUploadComponent extends VideoSend implements OnInit, OnDestroy
   }
 
   get videoExtensions () {
-    return this.serverConfig.video.file.extensions.join(', ')
+    return this.serverConfig.video.file.extensions.filter(e => e !== 'null').join(', ')
```

Comment:
Should be handled directly in `updateWebserverConfig` function in `constants.ts` so we don't include the `null` value in the object keys